receiver/sshcheckreceiver/metadata.yaml

```diff
@@ -0,0 +1,42 @@
+name: sshcheckreceiver
+
+resource_attributes:
+
+attributes:
+  ssh.endpoint:
+    description: Full SSH endpoint.
+    type: string
+
+metrics:
+  sshcheck.success:
+    description: The SSH server connected. 
+    enabled: true
+    gauge:
+      value_type: int
+    unit: 1
+    attributes: [ssh.endpoint]
+  sshcheck.duration:
```

Comment:
Based on @jmacd's [suggestion](https://github.com/open-telemetry/opentelemetry-collector-contrib/pull/14191#discussion_r987401449) for modeling a similar value, I plan to change this to a guage.